### PR TITLE
Update local-dev text to give accurate instructions.

### PR DESCRIPTION
**Pull request by danielhoherd**
## Description

Update instructions with modern kind syntax and also syntaxes that give the user instruction on how to log into astronomer root user account now that we have public signups disabled by default.

## Related Issues

N/A, just stuff I found while developing locally.

## Testing

This change modifies test scripts, so as long as tests pass then we're good with that. It would also be good to verify that the text at the end of the CI invocation of `bin/reset-local-dev` is correct. I've already verified it on my local machine.

## Merging

Merge everywhere.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ and set the pull policy to "Never".
 kubectl edit deployment -n astronomer <your deployment>
 ```
 
-#### Change Kubernetes version:
+#### Specify the Kubernetes version:
```
Comment (kwcanuck) [inline README.md:82]:
```suggestion
#### Specify the Kubernetes version
```

File: bin/reset-local-dev [hunk 2]
```diff
@@ -66,30 +66,33 @@ helm install -f "$CONFIG_FILE" \
   "$GIT_ROOT"
 
 
-echo '
-━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+cat <<EOF
+━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 
- Please run the following command to configure kubectl:
+  Astronomer is launching and should be available shortly. Run the following command
```
Comment (kwcanuck) [inline bin/reset-local-dev:72]:
```suggestion
 Run the following command to allow kubectl to access the Astro cluster:
```

File: bin/reset-local-dev [hunk 3]
```diff
@@ -66,30 +66,33 @@ helm install -f "$CONFIG_FILE" \
...
+  to configure kubectl to access the kind cluster that Astronomer is running in:
```
Comment (kwcanuck) [inline bin/reset-local-dev:73]:
```suggestion
  
```

File: bin/reset-local-dev [hunk 4]
```diff
@@ -66,30 +66,33 @@ helm install -f "$CONFIG_FILE" \
...
 
-     export KUBECONFIG="$(kind get kubeconfig-path --name=kind)"
+    kind export kubeconfig
 
-'
-echo "
- Astronomer is launching. You can check the status with:
+  You can then check the status with:
```
Comment (kwcanuck) [inline bin/reset-local-dev:77]:
```suggestion
  Run the following command to check the progress of the Astro installation:
```

File: bin/reset-local-dev (hunk 2 above)
Comment (kwcanuck) [reply bin/reset-local-dev:72]: I wasn't sure what you are referencing. Astronomer is a company, not a product offering. I assumed (perhaps incorrectly) Astro.

File: bin/reset-local-dev [hunk 5]
```diff
@@ -66,30 +66,33 @@ helm install -f "$CONFIG_FILE" \
...
 
-     kubectl get pods -n ${NAMESPACE}
+    kubectl get pods -n ${NAMESPACE}
 
+  After Astronomer is up (all pods running, all containers up), find the root login
```
Comment (kwcanuck) [inline bin/reset-local-dev:81]:
```suggestion
 Run the following command after the Astro installation is completed (all pods running and all containers available), to identify the root login credentials:
```

File: bin/reset-local-dev [hunk 6]
```diff
@@ -66,30 +66,33 @@ helm install -f "$CONFIG_FILE" \
...
+  credentials:
```
Comment (kwcanuck) [inline bin/reset-local-dev:82]:
```suggestion
  
```

File: bin/reset-local-dev [hunk 7]
```diff
@@ -66,30 +66,33 @@ helm install -f "$CONFIG_FILE" \
...
 
- After Astronomer is up (all pods running, all containers up), do this to start a connection to the service:
+    kubectl -n ${NAMESPACE} get secrets astronomer-root-admin-credentials \\
+      -o go-template --template='{{.data.password|base64decode}}{{\"\n\"}}'
 
-     sudo -E kubectl port-forward -n ${NAMESPACE} svc/astronomer-nginx 443
+  Next, port-forward into the kind cluster so you can access the application:
```
Comment (kwcanuck) [inline bin/reset-local-dev:87]:
```suggestion
 Run the following command to port-forward into the Astro cluster so you can access the application:
```

File: bin/reset-local-dev [hunk 8]
```diff
@@ -66,30 +66,33 @@ helm install -f "$CONFIG_FILE" \
...
 
+    sudo -E kubectl port-forward -n ${NAMESPACE} svc/astronomer-nginx 443
 
- Then you can access the service here: https://app.localtest.me
+  You can then access the service here: https://app.localtest.me
```
Comment (kwcanuck) [inline bin/reset-local-dev:91]: Is it a service, or an application? You're using the terms interchangeably. Select one term and use it consistently.

File: bin/reset-local-dev [hunk 9]
```diff
@@ -66,30 +66,33 @@ helm install -f "$CONFIG_FILE" \
...
 
- You can clean up like this:
+  Run the following command to remove everything:
```
Comment (kwcanuck) [inline bin/reset-local-dev:93]: What specifically are you removing? Instead of using _everything_, be specific.

**Review by kwcanuck — APPROVED** [review]

File: bin/reset-local-dev (hunk 2 above)
Comment (danielhoherd) [reply bin/reset-local-dev:72]: The repo is called "Astronomer", and for historical reasons so is the name of the software. Marketing calls it something else that is equally ambiguous.

File: bin/reset-local-dev (hunk 8 above)
Comment (danielhoherd) [reply bin/reset-local-dev:91]: In this context, the user is looking at a distributed system which constitutes an application. One of those components is a service that the user accesses to get the web UI, but that is not the only part of the application the user cares about.

**Pull request merged**